src/renderers/shaders/ShaderChunk/global_fragment.glsl

```diff
@@ -0,0 +1 @@
+// template for injecting varying, uniform and functions above main below other includes
```

Comment:
Given that you've implemented almost the exact same thing, could you please elaborate what you meant by abuse?